src/blocks/templates/blocks/section.html

```diff
@@ -0,0 +1,14 @@
+{% load wagtailcore_tags %}
+<div class="
+	    {% if value.padding == 'S' %} 
+	    padding-small 
+	    {% elif value.padding == 'M' %}
+	    padding-medium
+	    {% elif value.padding == 'L' %}
+	    padding-large
+	    {% endif %}
```

Comment:
These css classes does not seem to be defined. When I have a section with small padding, no css rule takes effect in my browser

Reply:
Good catch, I thought these were defined by materialize but I forgot that I added them myself in the previous PR, ill make a fix and redefine them.